src/components/Web3Status/index.tsx

```diff
@@ -215,9 +238,10 @@ export default function Web3Status() {
   const confirmed = sortedRecentTransactions.filter((tx) => tx.receipt).map((tx) => tx.hash)
 
   return (
-    <>
+    <span ref={ref}>
```

Comment:
I want the wallet to close when clicking outside of the container, this accomplishes that